src/core/server/workspaces/routes/index.ts

```diff
@@ -71,7 +69,7 @@ export function registerRoutes({
   );
   router.post(
     {
-      path: '/',
+      path: '',
```

Comment:
Is this change by intentional?

Reply:
Yes, update to follow RFC/generic HTTP convention

Reply:
Before: `/api/workspace/`
After: `/api/workspace`